src/classic/theme.js

```diff
@@ -178,6 +178,7 @@ function getTheme({ style, name }) {
       "panelInput.border": pick({ light: primer.gray[2], dark: primer.gray[1] }),
 
       "terminal.foreground": primer.gray[6],
+      "terminal.tab.activeBorder": "#f9826c",
```

Comment:
I can not reproduce this change:

![screenshot-Mon-19Jul21_01 15](https://user-images.githubusercontent.com/28626241/126086728-ab52f881-8a09-49cc-adad-dd7e6840dbb3.png)

I also can not find this attribute in the [VSCode Theme Color Documentation](https://code.visualstudio.com/api/references/theme-color). Is this a user setting?

Reply:
This is in Insiders only right now, coming in the next version

Reply:
Awesome, thank you for your contribution. can you please provide a link to the documentation for the insiders build in this PR comments so we can reference it? 🚀

Reply:
There's no documentation yet, here's where it's declared in code though:

https://github.com/microsoft/vscode/blob/7667e90bf82ece1395727c7d88a37114fe1488a4/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts#L40-L44

Reply:
> This is in Insiders only right now, coming in the next version

Thanks for clarifying. I think this should be fine to already merge. 👍